restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/UssdPushEndpoint.java

```diff
@@ -120,12 +120,13 @@ protected Response putCall(final String accountSid, final MultivaluedMap<String,
         final String to = data.getFirst("To");
         final String username = data.getFirst("Username");
         final String password = data.getFirst("Password");
-        final Integer timeout = getTimeout(data);
+        Integer timeout = getTimeout(data);
+        timeout = timeout != null ? timeout : 30;
```

Comment:
is this default 30 seconds delay used elsewhere? where is the RCML counterpart defiend? shouldnt come from conf?

Reply:
@jaimecasero i am improvising of existing implmentations, you can check this default 30 sec being used at following line 128.